_layouts/candidate.html

```diff
@@ -34,11 +34,10 @@ <h2>{{ candidate.name }}</h2>
             <div class="candidate__expenditure-ceiling">
               This candidate has agreed to the voluntary spending limits.
               The maximum contribution this candidate can accept is $800.
-              <a target="_blank" href="https://www.oaklandca.gov/services/boards-and-commissions-index/ethics/election-campaign-rules-and-disclosure"
-                 alt="Voluntary spending limits - more information">
-                <img src="{{ "/assets/images/icon_more_info.png" | relative_url }}"
+              <span class="hover-info-container"><img src="{{ "/assets/images/icon_more_info.png" | relative_url }}"
```

Comment:
How about we call this `help-info` and the inner content `help-info__content`? That follows our naming convention of `block__element--modifier`.